sixtyfps_compiler/layout.rs

```diff
@@ -227,14 +227,30 @@ impl LayoutConstraints {
     }
 
     pub fn visit_named_references(&mut self, visitor: &mut impl FnMut(&mut NamedReference)) {
-        self.max_width.as_mut().map(|e| visitor(&mut *e));
-        self.min_width.as_mut().map(|e| visitor(&mut *e));
-        self.max_height.as_mut().map(|e| visitor(&mut *e));
-        self.min_height.as_mut().map(|e| visitor(&mut *e));
-        self.preferred_width.as_mut().map(|e| visitor(&mut *e));
-        self.preferred_height.as_mut().map(|e| visitor(&mut *e));
-        self.horizontal_stretch.as_mut().map(|e| visitor(&mut *e));
-        self.vertical_stretch.as_mut().map(|e| visitor(&mut *e));
```

Comment:
/me liked that code much better before

Reply:
Yeap, that one indeed got more ugly!

I am actually surprised how little difference this makes in most places though! I had expected this particular lint to cause more ugliness in your code base (and I have never seen this `foo.map(||->() {...}` in any other project before.